prediction_market_agent/agents/microchain_agent/microchain_agent.py

```diff
@@ -138,7 +139,3 @@ def main(
 
 def get_editable_prompt_from_agent(agent: Agent) -> str:
     return extract_updatable_system_prompt(str(agent.system_prompt))
-
-
-if __name__ == "__main__":
```

Comment:
Was this deleted intentionally?

Reply:
Oops, it was originally but I will re-instate it